src/index.jsx

```diff
@@ -1,17 +1,60 @@
-import React from 'react';
-import { render } from 'react-dom';
-import './style.css';
+import React, { useState, useEffect } from "react";
+import { render } from "react-dom";
+import AnimalDetail from "./components/AnimalDetail/AnimalDetail";
+import AnimalList from "./components/AnimalList/AnimalList";
+import Search from "./components/Search/Search";
+import "./style.css";
 
 const App = () => {
+  const [animals, setAnimals] = useState([]);
+  const [animalDetail, setAnimalDetail] = useState(animals[0]);
+  const [zooParks, setZooParks] = useState(null);
+  const [inputValue, setInputValue] = useState("");
+
+  useEffect(() => {
+    fetch("https://lrolecek.github.io/zviratka-api/zvirata.json")
+      .then((response) => response.json())
+      .then((data) => {
+        setAnimals(data.zvirata);
+        setAnimalDetail(data.zvirata[0]);
+      })
+      .catch((err) => console.log(err.message));
+
+    fetch("https://lrolecek.github.io/zviratka-api/zoo.json")
+      .then((response) => response.json())
+      .then((data) => {
+        setZooParks(data.zoo);
+      })
+      .catch((err) => console.log(err.message));
+  }, []);
```

Comment:
kdybys chtěla, tak bonus 3:
- chtěl bych aby se na stránce zobrazil nějaký spinner (točítko) ve chvíli kdy data nebudou načtená
- tohle točítko bude čekat na načtení dat z obou endpointů.
- (jde to dělat více způsoby, zkus ale použít Promise.all)

Reply:
Jinak z praxe, typicky bys měla nějaký soubor "api", kde bys řešila takové ty implementační detaily jako "použije se fetch nebo axios?" či "vrátí se mi to v jsonu nebo ne?" a v komponentě už bys jen volala nějakou logickou funkci. Např:

```
  useEffect(() => {
    getAnimals().then(zvirata => {
        setAnimals(zvirata);
        setAnimalDetail(zvirata[0]);
    })
    .catch((err) => console.log(err.message));
    
   getAllZoo().then((zoos) => {
        setZooParks(zoos);
   })
      .catch((err) => console.log(err.message));
  }, []);
```
a tj pak api.js:
```
export const getAnimals = () => fetch("https://lrolecek.github.io/zviratka-api/zvirata.json")
      .then((response) => response.json())
      .then((data) => data.zvirata)
....
```